src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.test.tsx

```diff
@@ -233,6 +234,11 @@ describe('SavedObjectsTable', () => {
     }));
   });
 
+  afterEach(() => {
+    jest.clearAllMocks();
+    jest.resetModules();
```

Comment:
I am thinking maybe we should only clear the needed mocks instead of clearAll.

Reply:
It seems that we don't any afterEach code.

Reply:
Have deleted the logic